python/ray/autoscaler/commands.py

```diff
@@ -94,12 +94,16 @@ def request_resources(num_cpus=None, bundles=None):
         r.publish(AUTOSCALER_RESOURCE_REQUEST_CHANNEL, json.dumps(bundles))
 
 
-def create_or_update_cluster(
-        config_file: str, override_min_workers: Optional[int],
-        override_max_workers: Optional[int], no_restart: bool,
-        restart_only: bool, yes: bool, override_cluster_name: Optional[str],
-        no_config_cache: bool, dump_command_output: bool,
-        use_login_shells: bool) -> None:
+def create_or_update_cluster(config_file: str,
+                             override_min_workers: Optional[int],
```

Comment:
Is it too much to change this to `create_or_update_cluster(*, config_file: str....)`

Reply:
we should probably keep the hotfix minimal, but should have a discussion about these function signatures

Reply:
I think that'd break the product even more :)

Reply:
but yeah we should raise a warning saying that this will be done